eventually/static/src/containers/progress/Progress.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import LinearProgress from 'material-ui/LinearProgress';
+
+export const lightGreen500 = '#8bc34a';
```

Comment:
for what we export this constant?

Reply:
we can import all colors we need from material-ui
for example `import {blue500, yellow600, lime500} from 'material-ui/styles/colors';`

Reply:
done